website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Migrating Legacy Projects
+linkTitle: Migrating Legacy Projects
```

Comment:
```suggestion
linkTitle: Migrating Projects to 1.0.0+
```
"Legacy" is a moving target. Here we mean migrating to 1.0, but when 1.1 comes out, this could be misleading.

Reply:
@joelanford suggested we have not the 1.0.0 in another pr. I prefer we have the version as well. Let's see if he is ok @joelanford wdyt?

Reply:
Yeah this seems reasonable :+1: